engine/gamesys/proto/gamesys/buffer_ddf.proto

```diff
@@ -23,14 +23,14 @@ enum ValueType
 
 message StreamDesc
 {
-    required string name = 1;
+    required uint64 name_hash     = 1;
     required ValueType value_type = 2;
```

Comment:
Engine only data so should be fine, we don't use the string name any more, only the hash